src/Federation/Types/Any.php

```diff
@@ -14,9 +14,9 @@
  */
 class Any extends ScalarType
 {
-    public $name = '_Any';
+    public string $name = '_Any';
 
-    public $description = /** @lang Markdown */ <<<'DESCRIPTION'
+    public ?string $description = /** @lang Markdown */ <<<'DESCRIPTION'
```

Comment:
we should prob. standardize if we use `?string` or `string|null`

Reply:
Yes, when we make the switch to PHP 8.

Reply:
ah right, forgot it's php 8 😆